app/javascript/App.jsx

```diff
@@ -29,6 +31,7 @@ export default function App() {
   return (
     <>
       {currentUser?.signed_in && <Header /> }
+      {location?.pathname.startsWith('/rooms') && <BackgroundBuffer location={location.pathname} />}
```

Comment:
I thought about putting all the logic in <BackgroundBuffer>. And add a `return null` inside the component if the condition is not met. Wanted to stay consistent with the <Header> logic. IDK if it would impact the performance, but it might be a bit cleaner.

Reply:
I think its actually preferred that way - that way all of the logic is contained in one container

Reply:
All right I will put the hook back in